clients/src/main/java/org/apache/kafka/common/network/NetworkSend.java

```diff
@@ -23,20 +23,19 @@
  */
 public class NetworkSend extends ByteBufferSend {
 
-    public NetworkSend(String destination, ByteBuffer... buffers) {
-        super(destination, sizeDelimit(buffers));
+    public NetworkSend(String destination, ByteBuffer buffer) {
```

Comment:
I considered that, but I think simplicity/readability is more valuable than constructor consistency in this case. And, as you said, this way is more efficient too.